design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -0,0 +1,197 @@
+type Query{
+    # See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/framework/attributes-metadata.md
+    pageSpecificCustomAttributes(
+        page_type: CustomAttributesPageEnum!
+    ): CustomAttributeMetadata
+}
+
+type Mutation {
+    createReturn(input: CreateReturnInput!): CreateReturnOutput @doc(description: "Create a new return.")
+    addReturnComment(input: AddReturnCommentInput!): AddReturnCommentOutput @doc(description: "Add a comment to an existing return.")
+    addReturnTracking(input: AddReturnTrackingInput!): AddReturnTrackingOutput @doc(description: "Add tracking information to the return.")
+    removeReturnTracking(input: RemoveReturnTrackingInput!): RemoveReturnTrackingOutput @doc(description: "Remove tracking information from the return.")
+}
+
+input CreateReturnInput {
+    items: [CreateReturnItemInput]!
+    comment_text: String
+}
+
+input CreateReturnItemInput {
+    order_item_id: ID!
+    quantity_to_return: Float!
+    selected_custom_attributes: [ID!]
+    entered_custom_attributes: [EnteredCustomAttributeInput!]
+}
+
+input EnteredCustomAttributeInput {
+    id: ID!
+    value: String!
+}
+
+type CreateReturnOutput {
+    return: Return
+}
+
+input AddReturnCommentInput {
+    return_id: ID!
+    comment_text: String!
+}
+
+type AddReturnCommentOutput {
+    return: Return
+}
+
+input AddReturnTrackingInput {
+    return_id: ID!
+    carrier_id: ID!
+    tracking_number: String!
+}
+
+type AddReturnTrackingOutput {
+    return: Return
+}
+
+input RemoveReturnTrackingInput {
+    return_tracking_id: ID!
```

Comment:
This field's value is read by a client from `ReturnShippingTracking.id`. It's a bit hard to search for which type this ID comes from at the moment (I originally thought to look for `type ReturnTracking`. Can we rename either the field or the `ReturnShippingTracking` type, so it's clear the IDs are connected/related?

Reply:
Renamed the field.